src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,39 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_AMOUNT = 5;
+    private static final double MAGIC_NUMBER = 25;

```

Comment:
Please do not use MAGIC_NUMBER for a constant. 
Variable names should describe what the variable is used for.